controllers/csiscaleoperator_controller.go

```diff
@@ -229,9 +240,32 @@ func (r *CSIScaleOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	logger := csiLog.WithName("SetupWithManager")
 
+	p, err := predicate.LabelSelectorPredicate(metav1.LabelSelector{
+		MatchLabels: secretsLabels,
+	})
+	if err != nil {
+		return err // LOG: unable to create controller
```

Comment:
Please add a log before returning for debug purpose.

Reply:
Fixed. Added log message.